samba/docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: samba_homepage_1
+      APP_PORT: 9445
+
+  homepage:
+    image: joseluisq/static-web-server:2@sha256:c8345c2c469ba249cbc986b6155dc275ce86bcd30e0b29755fda4deb501bbddd
+    container_name: samba_homepage_1
+    environment:
+      - SERVER_ROOT=/var/public
+      - SERVER_PORT=9445
+    volumes:
+      - ${APP_DATA_DIR}/data/public:/var/public
+      - ${APP_DATA_DIR}/data/smb.conf:/var/public/smb.conf
+
+  server:
+    image: >-
+      dockurr/samba:4.20.1@sha256:9a57887c0be1242ee2e1381653619a656820e1aee8e271885662d04c2c367be8
+    container_name: samba_server_1
+    environment:
+      USER: umbrel
+      PASS: ${APP_PASSWORD}
```

To reduce the permissions, please change the environment variables to this
```suggestion
      PASS: ${APP_PASSWORD}
      UID: 1000
      GID: 1000
```
[Here](https://github.com/dockur/samba?tab=readme-ov-file#how-do-i-modify-the-permissions) is the explanation for that.